jobs/build-node-image.Jenkinsfile

```diff
@@ -174,12 +174,15 @@ lock(resource: "build-node-image") {
         currentBuild.result = 'FAILURE'
         throw e
     } finally {
+        message = ":openshift:"
         if (currentBuild.result == 'SUCCESS') {
             currentBuild.description = ":sparkles: ${build_description}"
```

Comment:
While we're here, can we switch back just the `currentBuild.description` ones (here and in the else block) to actual Unicode emojis? The ones used right now are Slack-native things and so don't render in the Jenkins UI.

Reply:
done!